portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,25 +23,42 @@
 import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+// @WebServlet("/data")
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String saying = "Hello Again";
-    ArrayList<String> messages = new ArrayList<String>();
-    messages.add("message one");
-    messages.add("message two");
-    messages.add("message three");
-    String json = convertToJsonUsingGson(messages);
-
-    response.setContentType("text/html;");
-    response.getWriter().println(json);
+//   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+//     String saying = "Hello Again";
+//     ArrayList<String> messages = new ArrayList<String>();
+//     messages.add("message one");
+//     messages.add("message two");
+//     messages.add("message three");
+//     String json = convertToJsonUsingGson(messages);
+
+//     response.setContentType("text/html;");
+//     response.getWriter().println(json);
+//   }
+
+//   private String convertToJsonUsingGson(ArrayList<String> mes) {
+//      Gson gson = new Gson();
+//     String json = gson.toJson(mes);
+//     return json;
+//   }
+
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      ArrayList<String> messages = new ArrayList<String>();
+      String text = getParameter(request, "comment", "");
+      messages.add(text);
```

Comment:
Same nit as here: https://github.com/hannahsalameh/step/pull/5#discussion_r432128405
Using fewer local variables can make code easier to read. E.g. `text` is used only once here.

Reply:
I have the variable here so that I can apply the function "getParameter()" to it. I am not sure how to do that without having a variable to except the return but it looks like everytime a comment is made this variable will be used to get the request... if I am not mistaken about this then I think it will be used a couple of times

Reply:
`doPost()` will be called a couple times, but within `doPost()`, the variable `text` is only used once. To avoid using a local variable, you can do something like:

message.add(getParameter(request, "comment", "");

Reply:
Okay, I'll change that!